.github/workflows/build-images.yaml

```diff
@@ -57,16 +56,14 @@ on:
         value: ${{ jobs.tags.outputs.cluster-autoscaler }}
 
 env:
-  IMG_CONTROLLER:         "neondatabase/neonvm-controller"
-  IMG_VXLAN_CONTROLLER:   "neondatabase/neonvm-vxlan-controller"
-  IMG_RUNNER:             "neondatabase/neonvm-runner"
-  IMG_KERNEL:             "neondatabase/vm-kernel"
-  IMG_SCHEDULER:          "neondatabase/autoscale-scheduler"
-  IMG_AUTOSCALER_AGENT:   "neondatabase/autoscaler-agent"
-  IMG_DAEMON:             "neondatabase/neonvm-daemon"
-  IMG_CLUSTER_AUTOSCALER: "neondatabase/cluster-autoscaler-neonvm"
-  ECR_DEV:                "369495373322.dkr.ecr.eu-central-1.amazonaws.com"
-  ECR_PROD:               "093970136003.dkr.ecr.eu-central-1.amazonaws.com"
+  IMG_CONTROLLER:         "neonvm-controller"
+  IMG_VXLAN_CONTROLLER:   "neonvm-vxlan-controller"
+  IMG_RUNNER:             "neonvm-runner"
+  IMG_SCHEDULER:          "autoscale-scheduler"
+  IMG_AUTOSCALER_AGENT:   "autoscaler-agent"
+  IMG_DAEMON:             "neonvm-daemon"
+  IMG_CLUSTER_AUTOSCALER: "cluster-autoscaler-neonvm"
+  GHCR_REGISTRY:          "ghcr.io/neondatabase"
```

Comment:
Are these changes still needed / actual?

I see the point and agree on idea to simplify workflows moving out "repetitive" parts as "neondatabase/" here.
Just we don't use much that further, and building `image-map` below we don't use any of these variables, but have static strings there:
https://github.com/neondatabase/autoscaling/pull/1283/files#diff-6ec9d6ffc2c8ebd369edcc698ad16020762b6616d45849d4b539557e59563884R393

Reply:
At this point we still need it.
As we move towards creating a multi-arch image using digest (for example, how we do it for [mutexbot image](https://github.com/neondatabase/dev-actions/blob/35a0807b8efa8f2aff2979d8d74dafb7f06a50f2/.github/workflows/mutexbot.yml#L122-L125)) instead of explicitly tagging it, we can get rid of the variable (as there'll be less usage of it)